widget/check.go

```diff
@@ -100,6 +100,31 @@ type Check struct {
 
 	focused bool
 	hovered bool
+
+	checkSource   binding.Bool
+	checkListener binding.DataListener
+}
+
+// Bind connects the specified data source to this Check.
+// The current value will be displayed and any changes in the data will cause the widget to update.
+// User interactions with this Check will set the value into the data source.
+//
+// Since: 2.0.0
+func (c *Check) Bind(data binding.Bool) {
```

Comment:
Should be defined lower down, with the rest of the methods, not above the constructors.

Reply:
I think that it is not above constructors so much as this file has not been updated to reflect the organisation - SetChecked, Hide, MouseIn, MouseOut, MouseMoved, Tapped, MinSize and CreateRenderer are all above the constructor.